tests/api/circulation/test_loan_utils.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests REST API for loans."""
+
+from copy import deepcopy
+
+import pytest
+from invenio_circulation.errors import RecordCannotBeRequestedError
+from utils import flush_index, item_record_to_a_specific_loan_state
+
+from rero_ils.modules.api import IlsRecordError
+from rero_ils.modules.items.utils import item_pid_to_object
+from rero_ils.modules.loans.api import Loan, LoanAction, LoanState, \
+    get_last_transaction_loc_for_item, get_loans_by_patron_pid
+from rero_ils.modules.loans.utils import can_be_requested
+from rero_ils.modules.locations.api import LocationsSearch
+
+
+def test_loan_utils(client, patron_martigny_no_email,
+                    patron2_martigny_no_email, circulation_policies,
+                    item_lib_martigny, librarian_martigny_no_email,
+                    loc_public_martigny):
+    """Test loan methods."""
```

Comment:
loan or loan utils methods?

Reply:
Is this file at the right place? tests/api is done for REST API tests. Moreover this is a test for a test function... not nice but ok.